### ISSUE TITLE
Harmonize VLAN table

### ISSUE THREAD

**Issue by ibot3** [agdsn/pycroft#348]
The `VLAN` table has currently the columns `id (PK)`, `name`, `vid`.
I think we want to have the `vid` column unique. That `id` differs from `vid` is confusing.

I would propose to use `vid` as a primary key and to remove the `id` column.

**Comment by mmarx** [comment]
VLAN ids are probably not globally unique, though.

**Comment by lukasjuhrich** [comment]
To be honest I don't find it too confusing, since (almost) everything has an integer primary key.
But this change does not have any impact on how the model is used, so simplicity is a plus, I guess.

Just make sure to have a reasonable schema downgrade that is a proper inverse to the upgrade (nontrivial).

**Comment by lukasjuhrich** [comment]
@mmarx that is true, but the NAT-Schema vlan information is stored elsewhere, and this table won't be affected by that. So VLANs in this relation will still be unique in the context in which they are used (dormitory-vlans). @sebschrader argued that „VLAN“ may be considered something globally unique, because technically VLAN and VLAN id are different concepts (a VLAN may be realized by different VLAN ids), but to avoid confusion, we will call the former a  `Layer2Domain` (will be equipped with an `isid`).
